src/main/scala/dpla/ingestion3/profiles/ProviderProfiles.scala

```diff
@@ -36,6 +36,15 @@ class EsdnProfile extends XmlProfile {
   override def getMapping = new EsdnMapping
 }
 
+/**
+  * Internet Archive
+  */
+class IaProfile extends JsonProfile {
+  type Mapping = CdlMapping // FIXME
```

Comment:
Do you want to keep `CdlMapping` on lines 43 and 46 for now?

Reply:
Yes, see above